openedx_webhooks/github_views.py

```diff
@@ -46,29 +49,34 @@ def pull_request():
         return "PONG"
 
     pr = event["pull_request"]
+    PR_NUMBER = pr['number']
```

Comment:
I changed it. But FYI: <https://en.wikipedia.org/wiki/Constant_(computer_programming)#Dynamically-valued_constants>.

Also, since Python seems to be taking a lot of cues from ES6: <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const>